<launch>
  <include file="$(find interactive_segmentation)/launch/organized_multi_plane_segmentation.launch" />
  <include file="$(find pcl_filter_utils)/launch/pcl_filter_utils.launch" />
  <include file="$(find point_cloud_image_creator)/launch/point_cloud_image_creator.launch" />
  <include file="$(find interactive_segmentation)/launch/edge_detector.launch" />
  <include file="$(find interactive_segmentation)/launch/supervoxel_segmentation.launch" />
  <node pkg="interactive_segmentation" name="interactive_segmentation"
	type="interactive_segmentation_node" output="screen">
    <remap from="input_image" to="/camera/rgb/image_rect_color" />
    <remap from="input_edge" to="/edge/image" />
    <remap from="input_cloud" to="/plane_extraction/output_nonplane_cloud" />

    <!--<remap from="input_image" to="/camera/rgb/image_rect_color" />
    <remap from="input_edge" to="/edge/image" />
    <remap from="input_cloud" to="/plane_extraction/output_nonplane_cloud" /> -->
  </node>
</launch>
